docs/changes.rst

```diff
@@ -4,6 +4,16 @@
 Changes
 #######
 
+Version 2.1
+===========
+
+Release 2.1.0
+-------------
+
+Features
+
+* Add optional field for thresholds.
```

Comment:
It might make sense to quote or otherwise highlight _optional_ here. Otherwise the obvious reading would be that some optional field was added, but who knows what it was. We want to convey that a field named "optional" was added.